Library/Homebrew/diagnostic.rb

```diff
@@ -507,7 +507,7 @@ def check_xdg_data_dirs
           this variable set to include other locations.
           Some programs like `vapigen` may not work correctly.
           Consider setting the XDG_DATA_DIRS for example like so
-              echo 'export XDG_DATA_DIRS="#{share}:$XDG_DATA_DIRS"' >> #{Utils::Shell.shell_profile}
+              echo 'export XDG_DATA_DIRS="#{share}:$XDG_DATA_DIRS"' >> #{Utils::Shell.profile}
```

Comment:
Apparently I also previously changed this on https://github.com/Linuxbrew/brew/commit/7709d182def0de573f142c749e4e20df9d64c8b2